ui/src/components/export/ExportForm.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import TextField from "@material-ui/core/TextField";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+
+class ExportForm extends React.Component {
+  render() {
+    return (
+      <Dialog
+        open={this.props.open}
+        onClose={this.props.handleClose}
+        aria-labelledby="form-dialog-title"
+      >
+        <DialogTitle id="form-dialog-title">Export to FireCloud</DialogTitle>
+        <DialogContent>
+          <DialogContentText>
+            Enter a workspace to continue in a FireCloud notebook.
```

Comment:
- Get rid of "Workspace Namespace" textbox. Let's copy https://bvdp-saturn-prod.appspot.com/#import-data and just have "Workspace Name".
- Get rid of sentence "Enter a workspace to continue in a FireCloud notebook.". We will not be continuing in a notebook.